src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx

```diff
@@ -69,6 +72,15 @@ PlusStatus vtkPlusWinProbeVideoSource::ReadConfiguration(vtkXMLDataElement* root
         }
     }
     
+    const char * fpdCharPtr = deviceConfig->GetAttribute("FocalPointDepth");
```

Comment:
Consider using the Plus xml macros. If one doesn't exist for a vector 4. feel free to create it.

Reply:
There is standard VTK method for this:  vtkXMLDataElement::GetVectorAttribute
https://www.vtk.org/doc/nightly/html/classvtkXMLDataElement.html#a3bd5f80f4cab1fde643409eca4bb7ea7